client/my-sites/email/titan-setup-thank-you/index.tsx

```diff
@@ -22,7 +22,7 @@ import { getSelectedDomain } from 'calypso/lib/domains';
 import { getDomainsBySite } from 'calypso/state/sites/domains/selectors';
 import { SiteData } from 'calypso/state/ui/selectors/site-data';
 import { ThankYou } from 'calypso/components/thank-you';
-import theme from 'calypso/my-sites/marketplace/theme';
+import theme from './theme';
```

Comment:
We usually use the full path for imports, even when importing from the same directory:

```suggestion
import theme from 'calypso/my-sites/email/titan-setup-thank-you/theme';
```

This makes it easier to look up those imports in an IDE or code editor. I'm also tempted to think that it also simplifies refactoring with [codemods](https://github.com/facebookarchive/codemod) (we used those a lot in the past).